src/System.Collections.Concurrent/src/System/Collections/Concurrent/BlockingCollection.cs

```diff
@@ -1059,22 +1093,24 @@ private static int TryAddToAnyCore(BlockingCollection<T>[] collections, T item,
         /// <returns>The index which the item has been added, -1 if failed</returns>
         private static int TryAddToAnyFast(BlockingCollection<T>[] collections, T item)
         {
-            for (int i = 0; i < collections.Length; i++)
+            for (int index = 0; index < collections.Length; index++)
```

Comment:
Why did you change this from i to index?  There's nothing in our guidelines against using i... it's very common and reasonable.